### ISSUE TITLE
Towards a cleaner WGLMakie look

### ISSUE THREAD

**Issue by piever** [JuliaPlots/WGLMakie.jl#106]
This is an umbrella issue to identify actionable items to bring the WGLMakie look closer to the clean CairoMakie svg. I imagine there are technical reasons why it can never be as polished, but I hope it is possible to reduce the gap.

Here I am making the same plot in WGLMakie and CairoMakie and comparing.

```julia
julia> using AlgebraOfGraphics

julia> set_aog_theme!()

julia> df = (x=rand(100), y=rand(100), z=rand(100));

julia> draw(data(df) * mapping(:x=>"gx", :y=>"gy", color=:z=>"group"));
```

WGLMakie output, after `using WGLMakie; WGLMakie.activate!()`:

<img width="671" alt="wglversion" src="https://user-images.githubusercontent.com/6333339/119644086-c8134580-be1c-11eb-8e1b-c237edb0c41c.png">

CairoMakie output, after `using CairoMakie; CairoMakie.activate!()`:

<img width="593" alt="cairoversion" src="https://user-images.githubusercontent.com/6333339/119644135-d5c8cb00-be1c-11eb-9943-c5d4fb4aca12.png">

Issues:

- Using a custom font (for instance `AlgebraOfGraphics.firasans("Medium")` and `AlgebraOfGraphics.firasans("Light")`, which are just paths to the ttf files) seems to mess up badly with font rendering. In the picture above some characters are rotated wrongly. When zoomming, some characters are missing, which seems to be #66. Question: is there a clean way to pass a custom font? I imagine maybe it could be possible to specify font-family and font-weight and let the browser figure it out.

- Resolution is a bit low, or there seems to be some artifacts. Question: is there a simple way to increase dpi or somehow fix the artifacts at the cost of performance?

- Removing spines seems to fail with WGLMakie. The `topspinevisible` and `rightspinevisible` keywords are ignored, same for the colorbar spine.

**Comment by SimonDanisch** [comment]
There is: https://github.com/JuliaPlots/WGLMakie.jl/pull/98

**Comment by SimonDanisch** [comment]
For the fonts, can you try rendering them a second time? There is this weird race condition that I'm fighting with forever, which doesn't update the font atlas correctly when adding new fonts and only displays them correctly after rendering them two times...
The spines one are weird, but maybe just a missing `visible` keyword forward (although I thought I fixed most of those).

**Comment by piever** [comment]
This can actually be closed already, as the linewidth as the spine and dpi issues were fixed in #109 and #110, and the font issue is tracked in #66
